data-access-analysis-tools/index-rsp-warnings.rst

```diff
@@ -12,98 +12,60 @@
 
+Data in /scratch is not backed up.
 
+Data in /projects is available for use in the situation where you wish to share files with other RSP users. For this data preview, any files in you put in /projects are readable and writeable by all users. Take care you do not delete other people's files.
 
-Butler-Related Warnings
------------------------
+For this data preview there are no file quotas applied, however usage will be monitored and users may be asked to reduce their footprint, either in their home space or shared data spaces. By using the Butler to write data, you are not using disk space but our data facility's object stores; ample capacity is available there and you should use that in favour of keeping large data files on disk.
 
-For DP0.1 or until we have a client/server Butler and signed URLs (DMTN-169,DMTN-176), all users share full read/write without restriction.  Since there is no individual access control, absolutely no use direct sql access will be allowed to any butler registry database.
-Delegates should only access the Butler repository using standard Butler APIs provided in the Rubin software stack.
+For this data preview, there are no write/delete restrictions on the shared user data Butler repository (butler-us-central1-dp01). That means you could delete your data and other user's data. Be very careful when using the Butler prune collection feature and in particular don't use wildcards with that command.
 
-User-added data in DP0.1’s butler repo will not be migrated to DP0.2’s butler repo. 
+You can reduce the risk of accidents by following the convention documented in `DMTN-167 <https://dmtn-167.lsst.io/>`__ and only write to your own ``u/<user>/*`` collections.
 
-Accessing data directly using the LSST Science Pipeline Butler interface: Limited processing capability will be available as part of DP0. All delegates will share access to a common Butler ('Gen3') repository.
-The butler repository is a shared read/write space for all delegates, and there is no per-user ownership of any data inside the butler repo. Therefore, it’s possible for one delegate to delete another delegate’s user-added datasets in the butler repo. For running pipetask with the DP0.1 repo, we strongly encourage delegates to follow the naming convention as documented in `DMTN-167 <https://dmtn-167.lsst.io/>` and only write to their own ``u/<user>/*`` collections. Although the convention is not enforced, it can reduce the chance of conflicts. We will **not** restore user-added data for delegates. Currently the butler repo is a shared-risk space and please plan for possible loss of user-added data. Project-loaded data are read only and protected.  We will address these issues in a future release before the operations starts.
+You should only access our Butler repositories using standard Butler APIs provided in the Rubin software stack. This is both to prevent accidents but also to fulfil your role in helping us evaluate our software.
```

Comment:
I don't think we want to delete that bit about not backing up user-added data do we? We only recently added that paragraph when it was decided that we didn't want to set up a huge backup policy for it (and CET signed off on this).

Reply:
+1.  We are not able to restore user-added data in the butler repo, no matter the accidents happen due to users errors, software bugs, or whatever.

Reply:
I only removed it while I checked with Hsinfang. It now reads:

"User data stored in our Butler repositories is not backed up, and may not be retained past the Data Preview 0.1 period."

Reply:
It seems the sentence got lost in the editing. I'm adding it in https://github.com/lsst/dp0-1_lsst_io/pull/27